### ISSUE TITLE
Duplicate "instance not found"

### ISSUE THREAD

**Issue by mffap** [zitadel/zitadel#7736]
### Preflight Checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions
- [ ] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Describe the docs your are missing or that are wrong

We
have https://zitadel.com/docs/self-hosting/deploy/troubleshooting
and 
https://zitadel.com/docs/self-hosting/manage/custom-domain#why-do-i-get-an-instance-not-found-error

Google/Search picks up only the first link (troubleshoot ZITADEL) and there is no reference to the  custom-domain page.

**Comment by muhlemmer** [comment]
Have you tried clicking this link?



![Image](https://github.com/zitadel/zitadel/assets/5411563/59fa6e8c-041b-47c3-a42b-4e5ba8b9658c)

**Comment by mffap** [comment]
> Have you tried clicking this link?

No, sorry! Ok link is there. Maybe we should just remove that info on the troubleshoot page, since it doesn't really add value.

**Comment by muhlemmer** [comment]
Well, the troubleshooting page is kind of pathetic with only one entry. But I actually asked in a review to split this a bit, as in my vision a troubleshooting page should be an overview of common problems with most-used fixes. Not much more then bullet points. Entries on the troubleshooting page then refer to reference documentation that explain the related concepts in more detail.

So for instance not found, the message from the troubleshooting page should be:

> You probably messed up the requested domain in the request or header vs what zitadel is configured with try x, y, z. > Here's a docs page on how zitadel uses domains in requests to learn more.

## potential example:

### Project roles are not returned in the userinfo

Roles can be returned in different ways:

- If you always want all roles from the current project, consider enabling the "Assert Project Roles"  in the [project role settings](https://zitadel.com/docs/guides/manage/console/projects#role-settings).
- If you want to assert if a user has one or more specific roles, use the `urn:zitadel:iam:org:project:role:{rolekey}` [reserved scope](https://zitadel.com/docs/apis/openidoauth/scopes#reserved-scopes) and replace `{roleKey}` with an actual role. Repeat for all the roles you wish to assert.
- If you want all roles from all projects in the requested audience, use the `urn:zitadel:iam:org:projects:roles` reserved scope in the auth request.

**Comment by elinashoko** [comment]
Thanks for filing this issue. As part of an ongoing backlog cleanup, we're closing issues that have had no activity in over 12 months.

If this is still relevant, let us know in a comment (or reopen the issue) and we'll take another look.

This issue will close in 30 days if there's no further activity.
